tests/js/spec/components/deprecatedforms/radioGroup.spec.tsx

```diff
@@ -0,0 +1,98 @@
+import {mountWithTheme, screen, userEvent} from 'sentry-test/reactTestingLibrary';
+
+import RadioGroup from 'sentry/components/forms/controls/radioGroup';
```

Comment:
Should this test module be in `deprecatedforms`?

Reply:
oh no... I think I missed this when rebasing